src/world/common/UnkFunc28.inc.c

```diff
@@ -0,0 +1,12 @@
+#include "common.h"
+#include "map.h"
+
+extern s16 gCurrentCamID;
+
+void N(UnkFunc28)(ScriptInstance *script, s32 isInitialCall) {
```

Comment:
Are you certain this is a script API func? The labelPositions below looks very weird and without `return clamp_angle` this is a no-op.

Reply:
Ok, this is an `onRender` for an Npc. i.e. the signature is `s32 N(UnkFunc28)(Npc* npc)` or something